texext/tests/test_plotdirective.py

```diff
@@ -27,10 +28,11 @@ def format_math_block(name, latex, label=None, number=None,
         number = 'True' if number is None else number
         label = 'True' if label is None else label
         id_part = '' if ids is None else 'ids="{}" '.format(ids)
+        nowrap = 'no-wrap="False" ' if SPHINX_ge_8p2 else ''
         return (
             '<math_block docname="{}" {}label="{}" '
-            'nowrap="False" number="{}" xml:space="preserve">{}'
-            '</math_block>'.format(name, id_part, label, number, latex)
+            '{}nowrap="False" number="{}" xml:space="preserve">{}'
```

Comment:
Sorry - just checking - is it really true that Sphinx >= 8.2 generates  something like`no-wrap="False" nowrap="False"` ... ?  Would you mind rewriting this as an f-string to it clearer what is going where?   Did you check the tests with Sphinx < 8.2?

Reply:
I did just check with Sphinx == 7 - it passes.

Reply:
OK - I need your fix for testing - so merging with my suggested edits - thanks.